configure.ac

```diff
@@ -408,6 +408,10 @@ fi
 AC_SUBST([AM_CFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Werror"])
 AC_SUBST([AM_CXXFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Wno-missing-braces -Werror"])
 
+#Add flags for SDL signoff called SDL325 - Compile With Defenses Enabled
+AC_SUBST([AM_CPPFLAGS], ["-O2 -D_FORTIFY_SOURCE=2 -fPIE -fPIC -fstack-protector"])
```

Comment:
`-O2 -fPIE -fPIC -fstack-protector` belong in `AM_CFLAGS` and `AM_CXXFLAGS`

Reply:
I have moved these flags into AM_CFLAGS and AM_CXXFLAGS, is it OK?